site/content/docs/cli/constructs-reference.md

```diff
@@ -171,6 +170,7 @@ for rendering the body type in the web UI and not needed in most cases using the
 - `headers`: An array of `{ key: 'X-My-Header', value: 123 }` objects to define HTTP headers.
 - `queryParameters`: An array of `{ key: 'my-param', value: 123 }` objects to define query parameters.
 - `followRedirects`: A boolean indicating automatic following of any `30x` redirects.
+- `shouldFail`: A boolean that indicates whether the request should fail.  This allows you to treat HTTP error codes (4xx and 5xx) as correct responses.
```

Comment:
`shouldFail` is at the `Check` level, not `Request`. Also, it only works for API checks. This is a consequence of our early API and less than ideal, but it is how it works now.

Reply:
Adding this back to the Check level, and I'll stipulate that it only works for API checks.